avail-server/src/main/kotlin/com/avail/server/io/ServerInputChannel.kt

```diff
@@ -150,7 +150,8 @@ class ServerInputChannel constructor(
 			markBuffer = null
 			if (waiters.isEmpty())
 			{
-				val message = Message(buffer.toString())
+				val message = Message(
+					buffer.toString().toByteArray(), channel.state)
```

Comment:
@toddATavail I recall this feeling a bit weird. I'm going to look at it again...

Reply:
Never mind, these are all CharBuffers that need to be made into Strings before they can be made into ByteArrays.